ballerina/tests/test.bal

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/time;
+import ballerina/os;
+
+configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
+configurable string userId = isLiveServer ? os:getEnv("TWITTER_USER_ID") : "test";
+configurable string token = isLiveServer ? os:getEnv("TWITTER_TOKEN") : "test";
+configurable string serviceUrl = isLiveServer ? "https://api.twitter.com/2" : "http://localhost:9090/";
+
+ConnectionConfig config = {auth: {token: token}};
+final Client twitter = check new Client(config, serviceUrl);
+
+//Test Posting a Tweet
```

Comment:
No need to comment the test cases. The test case name should indicate what it does. Check for other places too.

Reply:
[Update mock_service and test bal files with by removing unecessory comments and empty lines](https://github.com/ballerina-platform/module-ballerinax-twitter/pull/112/commits/aa484ffa9cce3ffe931d44e1c0ecebc588938a67) 
[aa484ff](https://github.com/ballerina-platform/module-ballerinax-twitter/pull/112/commits/aa484ffa9cce3ffe931d44e1c0ecebc588938a67)